### DISCUSSION TITLE
Customize Experience

### DISCUSSION THREAD

**Discussion by Bakadori** [builtbybel/Bloatynosy#257] [General]
Hello!

I find it a bit confusing that after checking various items I need to hit the Enable All button to disable those checked, and the Disable All if I want to enable them again. Wouldn't something like Apply Selected and Revert Selected would be more intuitive names for those buttons? Or have I completely misunderstood the whole thing?

In any case, thanks for putting this together.

**Comment by Belim** [comment]
Hi, that's not the intention at all, and you're misunderstanding it 😨  Either you adjust individual settings directly using the checkboxes, or you click on 'Enable All,' which will activate all the checkboxes, or 'Disable All,' which will deactivate them

**Reply by Bakadori** [reply]
Doh, I was just using it in another computer and realized that 😅. Anyway, maybe it's just me, but I find the Enable the Disable and Disable Enabled Disables a bit confusing. Maybe Activate All, Deactivate All and Activated / Deactivated in grey? Or probably it's just me being a bit obtuse. Anyway, I (think that) I get it now. Thanks for your reply and the great tool!

**Reply by Belim** [reply]
you re right, 'activate' does sound more fitting than 'enable' for now. i'll switch it 😸
